core/datainput.gms

```diff
@@ -1008,25 +1008,32 @@ $ENDIF.WindOff
 *** pm_conv_cap_2_MioLDV <- 650  # The world has slightly below 800million cars in 2005 (IEA TECO2), so with a global vm_cap of 1.2, this gives ~650
 *** ==> 1TW power plant ~ 650 million LDV
 
-  p_adj_coeff(ttot,regi,te)                = 0.2;
+  p_adj_coeff(ttot,regi,te)                = 0.25;
   p_adj_coeff(ttot,regi,"pc")              = 0.5;
-  p_adj_coeff(ttot,regi,"ngcc")            = 0.5;
+  p_adj_coeff(ttot,regi,"ngcc")            = 0.4;
   p_adj_coeff(ttot,regi,"igcc")            = 0.5;
+  p_adj_coeff(ttot,regi,"bioigcc")         = 0.55;
+  p_adj_coeff(ttot,regi,"gaschp")          = 0.4;
+  p_adj_coeff(ttot,regi,"coalchp")         = 0.5;
+  p_adj_coeff(ttot,regi,"biochp")          = 0.55;
   p_adj_coeff(ttot,regi,"coaltr")          = 0.1;
   p_adj_coeff(ttot,regi,"tnrs")            = 1.0;
   p_adj_coeff(ttot,regi,"hydro")           = 1.0;
-  p_adj_coeff(ttot,regi,teCCS)             = 1.0;
   p_adj_coeff(ttot,regi,"gasftrec")        = 0.4;
-  p_adj_coeff(ttot,regi,"gasftcrec")       = 0.8;
   p_adj_coeff(ttot,regi,"coalftrec")       = 0.6;
-  p_adj_coeff(ttot,regi,"coalftcrec")      = 0.8;
-  p_adj_coeff(ttot,regi,"spv")             = 0.08;
-  p_adj_coeff(ttot,regi,"wind")            = 0.15;
-  p_adj_coeff(ttot,regi,"geohe")            = 0.6;
+  p_adj_coeff(ttot,regi,"bioftrec")        = 0.65;
+  p_adj_coeff(ttot,regi,"gash2")           = 0.35;
+  p_adj_coeff(ttot,regi,"coalh2")          = 0.55;
+  p_adj_coeff(ttot,regi,"bioh2")           = 0.6;
+  p_adj_coeff(ttot,regi,teCCS)             = 1.0;
```

Comment:
`coalftcrec` is part of `teCCS`. 
As `teCCS` definition is now written after the `coalftcrec` definition, you are redefining its value from 0.6 to 1.
If this is not intentional, i.e. `coalftcrec` should be equal to 0.6, please move the `teCCS` line above `coalftcrec ` definition. If it is intentional, please remove the `coalftcrec` definition to avoid confusion.

PS: I didn't checked if this also happens with other techs.